commit 58331306f064daca44d249201adbbfe7f156bedc
Author: Dave Eddy <dave@daveeddy.com>
Date:   2019-05-15T14:20:57-04:00 (5 months ago)
    
    MANTA-4279 fix make check for boray, add back in rust checks

diff --git a/Makefile b/Makefile
index b514038..b84b440 100644
--- a/Makefile
+++ b/Makefile
@@ -4,6 +4,8 @@
 
 NAME = manta-boray
 
+RUST_CODE = 1
+
 SMF_MANIFESTS_IN = smf/manifests/boray.xml.in
 
 ENGBLD_USE_BUILDIMAGE =	true
@@ -82,11 +84,6 @@ publish: release
 build-boray:
 	$(CARGO) build --release
 
-.PHONY: rust-check
-rust-check:
-	cargo check
-
-
 include ./deps/eng/tools/mk/Makefile.deps
 include ./deps/eng/tools/mk/Makefile.agent_prebuilt.targ
 include ./deps/eng/tools/mk/Makefile.smf.targ
diff --git a/deps/eng b/deps/eng
index dcf15a6..d25b8fc 160000
--- a/deps/eng
+++ b/deps/eng
@@ -1 +1 @@
-Subproject commit dcf15a6507556313d7925c6914c0040fbc52161e
+Subproject commit d25b8fc60fb8c649559924870fe3aaf75e7421d5
